samples/sdfw/ssf_client/Kconfig

```diff
@@ -16,11 +16,6 @@ config ENABLE_ECHO_REQUEST
 	depends on SSF_ECHO_SERVICE_ENABLED
 	default y
 
-config ENABLE_PRNG_REQUEST
-	bool "Enable the PRNG request service"
-	depends on SSF_PRNG_SERVICE_ENABLED
-	default y
-
```

Comment:
Should delete `PRNG_BYTES_TO_GET` below as well I think

Reply:
done